### DISCUSSION TITLE
Orchard Developers for hire?

### DISCUSSION THREAD

**Discussion by LivyChops** [OrchardCMS/OrchardCore#12183] [General]
Hi All, Is there a list or pinned board of Orchard Developers for hire anywhere?

I created an Orchard 1 site for a not-for-profit I'm involved with some time ago; which, unfortunately, was scrapped by the people who took over from me for being too slow (laughably, no site has existed since then).
Anyhow, as a glutton for punishment, I have volunteered my time again, and wish to start afresh with Core. But knowing how long it took me to deliver something useable the first time, I know my skills lie in graphics rather than coding.

The best way I can think to describe the project quickly, is a real estate listing site (as they encompass 95% of the major aspects we're after - as in, the user needs to be able to: REGISTER as a user/MEMBER | LIST an item/s with description and imagery | that are placed on a MAP for others to search | with a CALENDAR for our regular work bees.

I believe I have three options with the funds we've raised:

1. Pay an experienced developer for an entire bare-bones project they've previously completed that 'somewhat' fits the bill. (I have no idea of costs - thoughts/approximate cost?)
2. I try and do it all again, which will be a bad investment in 'time' resulting in poor productivity, especially when it's irregular volunteered hours.
3. Try and cobble together as many existing modules and Pay someone intermittently to complete any tasks/module gaps as they arise.

Either way, at some point, we're going to need to involve an expert; hence, can anyone point me toward some experienced Orchard developers? …oh, and any thoughts/suggestions/advice will be greatly appreciated :)

Many thanks - and apologies for the lengthy post. Liv

**Comment by hishamco** [comment]
There's no list for that AFAIK, but it's good idea to check Gitter or creating a list here, I'm suggesting to consult specialiest companies such as Lombiq (@Piedone) or any Orchard Core freelancer might find this interested

**Reply by Piedone** [reply]
Thanks for the shoutout, Hisham! Yeah, @LivyChops, if you write to us at Lombiq under https://lombiq.com/contact-us we're happy to check out what you need and discuss options. BTW one of our long-time clients has a very similar project, perhaps there's some opportunity for collaboration.

**Comment by Piedone** [comment]
This is probably the place to bring this up :).

I'd advise against doing option 3: That's where you try to do all the groundwork instead of letting an experienced OC developer do it, and then hand over a perhaps less than ideal implementation to one to sort out. This is the worst of both worlds.

You say you have graphics skills, and you know the client. So, in your place, I wouldn't do option 2 either, rather, I'd focus on these strengths, and explore the requirements with the client, do the graphic design of the app, and create interactive design specs with it (think InVision or similar). This is very valuable work, and something that needs to be done before any significant development can commence (not necessarily for the full app in a waterfall way, but for basics, and then features/sections continuously). It also reduces the costs of development, since it's not the OC developer who has to figure out details with the client (it's a question though, whether your time or the developer's is more expensive).

So, if the budget allows, I suggest you focus on the requirements and design, and hand over development completely to an experienced OC developer. If it doesn't, then there's also option 4: You do the requirements and design, then let an OC developer do the groundwork for the new app (this is where their experience can provide the most value), and then you take over to develop smaller, specific parts (like the styling for the rest of the pages, but following the example of the theme foundations).

**Reply by LivyChops** [reply]
@Piedone Thank you so much for your very insightful and considered reply Zoltan, I sincerely appreciate your effort in dissecting my post/project case, so thoroughly.

All of your proposed methods are much more enticing than me starting all over again, and 'yes', whether it’s a bare-bones deliverable; a collaboration with your long-time client; etc., - I'm definitely keen to play to my design strengths like you say 'more so' than anything else (…if budget allows that in its entirety, that would be fantastic); so quite frankly, I will be emailing you at Lombiq tomorrow.

Thanks again, and I look very much forward to your future solution.

**Reply by Piedone** [reply]
Looking forward to it!
